src/app/modules/consultations/consultation-profile/consultation-public-responses/consultation-public-responses.component.ts

```diff
@@ -77,4 +82,22 @@ export class ConsultationPublicResponsesComponent implements OnInit, AfterViewCh
     });
   }
 
+  getActiveRound(profileData) {
+    if (profileData && profileData.responseRounds) {
+      const responseRounds = profileData.responseRounds;
+      if (responseRounds && responseRounds.length) {
+        const activeRound  = responseRounds.find((round) => round.active);
+        if (!isObjectEmpty(activeRound)) {
+          return activeRound.roundNumber;
+        }
+      }
+    }
+    return;
+  }
+
+  setActiveRound(roundNumber) {
```

Comment:
@Nishanthh Is this being used? I can see line 46 already doing what this function is supposed to do

Reply:
@avikt in changing the round, to update we round number we need this function bro.